### DISCUSSION TITLE
State of GHelper without Asus drivers

### DISCUSSION THREAD

**Discussion by ghost** [seerge/g-helper#2518] [General]
It seems like when Asus drivers are missing, GHelper is unable to show any details about the GPU or either of the GPU/CPU fans. It can still read the CPU temperature and (apparently) set CPU "modes". No keybindings work and the keyboard backlight controls don't work either. Visual modes also do not work.

I am running a hardened version of windows so installing the Asus drivers isn't the best option for me. I was just wondering if there's any hope for restoring more functionality?

**Comment by seerge** [comment]
@thepriyamkalra you need Asus System Control Interface to be able to control your device 

It’s clearly mentioned in the readme
https://github.com/seerge/g-helper/wiki/Requirements#requirements-mandatory

**Reply by ghost** [reply]
Older versions used to give an error when it was absent; not seeing that, I thought something changed. Thanks!

**Reply by ghost** [reply]
I just realised that somehow the keyboard backlight and some other keybinds work on my linux installation! I have not installed any asus drivers here either; so there maybe some way to make them work with GHelper too (without depending on ASUS drivers). Let me know if you're interested.

**Reply by seerge** [reply]
@thepriyamkalra Linux has own driver / way to access hardware layer. 

In windows this driver already exists, and it’s official from Asus :) I don’t plan to write own drivers atm :)
